### PR TITLE
Add vte

### PR DESCRIPTION
**Pull request by satmandu**
- this is a gnome terminal dep
 
Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

File: packages/vte.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+require 'package'
+
+class Vte < Package
+  description 'Virtual Terminal Emulator widget for use with GTK'
```
Comment (uberhacker) [inline packages/vte.rb:4]: Missing homepage?

File: packages/vte.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  })
+  binary_sha256 ({
+     aarch64: '948e60479e834c4cf9c3fdebab5795faf64c38122d4662f4c355b1068078f743',
+      armv7l: '948e60479e834c4cf9c3fdebab5795faf64c38122d4662f4c355b1068078f743',
+        i686: '88d7d0d1a8b3e8b86a8ff3c66aa6fc7b1ae8d57712d1659c674bc3b129fd2fcd',
+      x86_64: 'f261f381d4ff58761907737037ed60ddafff5444067b38fb28e45f02f04ed6aa',
+  })
+
+  depends_on 'gobject_introspection' => ':build'
+  depends_on 'icu4c'
```
Comment (uberhacker) [inline packages/vte.rb:24]: This is a core package so no need to list here.

File: packages/vte.rb (hunk 1 above)
Comment (satmandu) [reply packages/vte.rb:4]: fixed

File: packages/vte.rb (hunk 2 above)
Comment (satmandu) [reply packages/vte.rb]: fixed

**Pull request merged**